module/VuFind/src/VuFind/I18n/Translator/Loader/ExtendedIni.php

```diff
@@ -37,218 +38,209 @@
  * @category VuFind
  * @package  Translator
  * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Sebastian Kehr <kehr@ub.uni-leipzig.de>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Site
  */
-class ExtendedIni implements FileLoaderInterface
+class ExtendedIni implements RemoteLoaderInterface
```

Comment:
We now implement `RemoteLoaderInterface` for the sake of better abstraction as the main [`load`](https://github.com/vufind-org/vufind/blob/c5ec6f98ac86daef4200b50d3d65bbbd495371d4/module/VuFind/src/VuFind/I18n/Translator/Loader/ExtendedIni.php#L128-L155) method only receives the locale and text domain.